### DISCUSSION TITLE
192

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#165] [General]
# 潮流周刊第192期 - 青皮橘子

封面图拍摄于小红书上买到的青皮橘子，好多年没有吃过这种绿中透黄的酸甜口了，大家是不是也很久没有遇到了

https://weekly.tw93.fun/posts/192-%E9%9D%92%E7%9A%AE%E6%A9%98%E5%AD%90

**Comment by Vainycos** [comment]
文末抛出的新颖的观点引发了我的深思

**Comment by zcyc** [comment]
建议试试 Vivaldi Browser

**Reply by tw93** [reply]
哈哈 我之前用过好几次，的确这个浏览器不是特别好看的样子

**Reply by zcyc** [reply]
> 哈哈 我之前用过好几次，的确这个浏览器不是特别好看的样子

哈哈，你可是 Frontend Engineer，以你的审美和技术，让它变好看就是分分钟的事。之前看你折腾过 Edge，觉得 Vivaldi 太适合你了。

**Comment by zimiyu** [comment]
Telegram禁用Telegraph上传功能，Telegraph-Image已经贵啦

**Reply by tw93** [reply]
哈哈 那就很建议直接oss了

**Comment by JasonSubMara** [comment]
<少读点书，多刷刷抖音吧>

里面的内容蛮有意思的，不过好像按照他的理论我的号废了，因为全是爱看的🦵
